### PR TITLE
CAEN linux overhaul

### PR DESCRIPTION
**Pull request by awdeorio**
Working on #159 reminded me that this tutorial hasn't seen an update in a long time.  I freshened it up, shortening some stuff and adding reminders about the UM VPN.

Q: Do you think we should add screenshots of how to confirm you're on the VPN?

### CODE REVIEW

**Comment by awdeorio** [comment]
Brain wave: should we link from the CLI tutorial to the this tutorial for SSH and rsync advice?

File: docs/setup_caen.md [hunk 1]
```diff
@@ -12,26 +12,27 @@ This tutorial will show you how to copy source code from your Laptop to CAEN Lin
 
 <img src="images/caen005.excalidraw.png" width="768px" class="invert-colors-in-dark-mode" />
 
-## Prerequisites
-You have installed `ssh` and `rsync`.  Your versions might be different.
+## Install
+Everyone who registers for an EECS class (like EECS 280) should receive a CAEN account automatically by the first day class.  If you register after the first day of class, you should get your account within 24 hours of registration.
+
+### `ssh` and `rsync`
+Make sure you have `ssh` and `rsync` installed.  Your versions might be different.  They are install by default on macOS and Windows/WSL users may have already followed [these instructions](setup_wsl.html#install-cli-tools).
```
Comment (amirkamil) [inline docs/setup_caen.md:19]: s/install/installed/

**Review by amirkamil — APPROVED** [review]
LGTM

**Comment by awdeorio** [comment]
I'm going to wait to merge this because CAEN staff wanted to take a look and make suggestions.

**Comment by awdeorio** [comment]
I pushed a couple more suggestions after an live discussion with CAEN.

**Pull request merged**